src/huggingface_hub/inference/_client.py

```diff
@@ -1309,3 +1310,50 @@ def _resolve_url(self, model: Optional[str] = None, task: Optional[str] = None)
             # Otherwise, we use the default endpoint
             else f"{INFERENCE_ENDPOINT}/models/{model}"
         )
+
+    def get_model_status(self, model: Optional[str] = None, *, token: Optional[str] = None) -> ModelStatus:
+        """
+            A function which returns the status of a specific model, from the Inference API.
+
+            Args:
+                model (`str`, *optional*):
+                    Identifier of the model for witch the status gonna be returned(retrieve). If model is not provided, 
+                    the model associated with this instance of InferenceClient will be used. The 
+                    identifier should not be a URL.
+
+                token (`str`, *optional*)
+                     Optional token for authenticating with the Hugging Face API. If not provided, the 
+                     token associated with this instance of InferenceClient will be used.
+            
+
+            Returns:
+                ModelStatus: An instance of ModelStatus dataclass, containing information,
```

Comment:
```suggestion
                [`ModelStatus`]: An instance of ModelStatus dataclass, containing information,
```

With this syntax, the generated docs will create a link when you click on it

Reply:
Oh it's perfect. I didn't know that there is such functionality. Very interesting. Ok now I see why there is such way to write comments. Nice :smile: